### ISSUE TITLE
Button support for Logitech M720 Triathlon. [enhancement]

### ISSUE THREAD

**Issue by aliyilma** [noah-nuebling/Mac-Mouse-Fix#1214]
## Description
Thanks for the excellent app! I'm using it with pleasure. However, I can't process some buttons on my M720 mouse, can there be an update for these?

- I think the center button/wheel tilt left and right are separate buttons. In some specific applications (e.g. Blender) they have their own functions.
- There is a -kind of- hidden button towards the bottom where the thumb rests. By default this button seems to cover the three/four finger push up gesture on the trackpad.
For now I can't customize these mouse buttons in the app. Also app-specific settings would be awesome.

## Rationale
- Button customization
- App-specific settings

## Additional info
–

**Comment by lutek** [comment]
I agree that the left and right tilt buttons are separate. The buttons are ideal for moving Space.

**Comment by chri50** [comment]
I agree - i would like to use the Logitech M720 as my new default mouse and customizing the thumb button would be great!
thx!

**Comment by westernwna** [comment]
Side Scroll aka Wheel Tilt Left/Right for moving spaces will be really nice.
and will be more nicer if thumb button of M720 can be programmable.

Actually that button is programmable in Logi+ Option app.
but I completely remove that option app and use this Mac mouse fix app.
Really appreciate for all the hard work to create such an amazing app.
Thank you.

**Comment by Skeeve** [comment]
I came here because "SensibleSideButtons" has neither a wheel click support, nor support for tilt. Unfortunately no tilt support here as well. I agree with the others that this would be great for changing desks or spaces.
